### PR TITLE
Benchmarking documentation (Lombiq Technologies: OCORE-170)

### PR DESCRIPTION
**Pull request by Piedone**
As we discussed on today's meeting. Just some rudimentary docs and a link to https://aka.ms/aspnet/benchmarks.

### CODE REVIEW

File: src/docs/guides/contributing/benchmarking.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Benchmarking
+
+To measure how fast Orchard Core is (usually, it's pretty fast :)), we employ some benchmarking. You can help extend the benchmarking suite too! (See the [code contribution guidelines](contributing-code.md).)
+
+## `OrchardCore.Benchmarks`
+
+In the [`OrchardCore.Benchmarks` project](https://github.com/OrchardCMS/OrchardCore/tree/main/test/OrchardCore.Benchmarks) we have several benchmarks created with [BenchmarkDotNet](https://benchmarkdotnet.org/). When we troubleshoot some performance issue, we extend these to measure the performance impact of the changes.
```
Comment (hishamco) [inline src/docs/guides/contributing/benchmarking.md:7]:
```suggestion
In the [`OrchardCore.Benchmarks` project](https://github.com/OrchardCMS/OrchardCore/tree/main/test/OrchardCore.Benchmarks) we have several benchmarks created with [BenchmarkDotNet](https://benchmarkdotnet.org/). When we troubleshoot some performance issues, we extend these to measure the performance impact of the changes.
```

**Review by hishamco — CHANGES_REQUESTED** [review]

**Comment by hishamco** [comment]
Can we document something about writing benchmarks in `OrchardCore.Benchmarks`?

**Comment by Piedone** [comment]
Surely, but having never written one, I'm not the guy for that.

**Review by MikeAlhayek — APPROVED** [review]

**Comment by hishamco** [comment]
I already did :) but let us merge this for now

**Review by hishamco — APPROVED** [review]

**Pull request merged**